CHANGELOG.md

```diff
@@ -26,12 +26,14 @@ Date: 01/19/2021
 - [[PR 386]](https://github.com/lanl/parthenon/pull/386) Introduce `Private`, `Provides`, `Requires`, and `Overridable` variable metadata, allowing fine-grained control of conflict resolution between packages.
 
 ### Changed (changing behavior/API/variables/...)
+- [[PR 425]](https://github.com/lanl/parthenon/pull/425) Remove ambiguity in package names. `Packages_t` no longer has an `operator[]` method. This has been replaced with `Add` and `Get`.
```

Comment:
You need to move this up to the `Current develop` section.

Reply:
Oops. Thanks for the catch. That must have somehow gotten caught in the merge after the release.

Reply:
Done.